packages/autorest.python/test/vanilla/legacy/AcceptanceTests/asynctests/test_header.py

```diff
@@ -122,7 +122,7 @@ async def test_string(self, client, value_header):
         response = await client.header.response_string("null", cls=value_header)
         assert response == "null"  # TODO This should be None
         response = await client.header.response_string("empty", cls=value_header)
-        assert response == ""
+        assert response is None
```

Comment:
`RestAioHttpTransportResponse` uses [CIMultiDict](https://github.com/Azure/azure-sdk-for-python/blob/0a0a6a3214002563cd1d5c2e3ba004ce5efa18ef/sdk/core/azure-core/azure/core/rest/_aiohttp.py#L175) which has special logic for [get](https://github.com/Azure/azure-sdk-for-python/blob/0a0a6a3214002563cd1d5c2e3ba004ce5efa18ef/sdk/core/azure-core/azure/core/rest/_aiohttp.py#L135-L139):
![image](https://github.com/Azure/autorest.python/assets/70930885/eb04b9ec-a25c-4cb3-9817-8da939e590e6)

result is different with sync http response: https://github.com/Azure/autorest.python/blob/7370ed827c3da3207557e9036a3bef456dbe6b94/packages/autorest.python/test/vanilla/legacy/AcceptanceTests/test_header.py#L133

Is it expected?

Reply:
i think this is fair, it's a limitation of the aiohttp transport